internal/webapi/server.go

```diff
@@ -114,9 +113,17 @@ func (a *Api) Authenticate(auth authentication.Authenticator[APIAuthCredientals]
 		return nil, err
 	}
 
-	headers := make(map[string]string)
-	headers["Set-Cookie"] = makeAuthCookie(token)
-	return &femto.EmptyBodyResponse{Headers: headers, Status: http.StatusAccepted}, nil
+	cookies := []http.Cookie{{
+		Name:     authentication.TOKEN_KEY,
+		Value:    token,
+		Path:     "/",
+		MaxAge:   3600,
+		HttpOnly: true,
+		Secure:   true,
+		SameSite: http.SameSiteLaxMode,
```

Comment:
This used to be Strict in the old version, now it's lax. Is this Ok??

Reply:
No idea, didn't mean to change it and I think it should be fine as strict, so changed in back.